x-pack/plugins/fleet/server/routes/package_policy/handlers.ts

```diff
@@ -173,15 +187,30 @@ export const createPackagePolicyHandler: FleetRequestHandler<
   const soClient = fleetContext.epm.internalSoClient;
   const esClient = coreContext.elasticsearch.client.asInternalUser;
   const user = appContextService.getSecurity()?.authc.getCurrentUser(request) || undefined;
-  const { force, ...newPolicy } = request.body;
-  // TODO Remove deprecated APIs https://github.com/elastic/kibana/issues/121485
-  delete newPolicy.output_id;
+  const { force, package: pkg, ...newPolicy } = request.body;
+  if ('output_id' in newPolicy) {
+    // TODO Remove deprecated APIs https://github.com/elastic/kibana/issues/121485
+    delete newPolicy.output_id;
+  }
   const spaceId = fleetContext.spaceId;
   try {
-    const newPackagePolicy = await packagePolicyService.enrichPolicyWithDefaultsFromPackage(
-      soClient,
-      newPolicy as NewPackagePolicy
-    );
+    let newPackagePolicy: NewPackagePolicy;
+    if (isSimplifiedCreatePackagePolicyRequest(newPolicy)) {
+      if (!pkg) {
+        throw new Error('Package is mandatory');
```

Comment:
(nit) I think `Package is required` is more in line with other error messages we throw elsewhere.